src/Block/Service/EmptyBlockService.php

```diff
@@ -18,22 +18,11 @@
 use Sonata\BlockBundle\Model\BlockInterface;
 use Sonata\CoreBundle\Validator\ErrorElement;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class EmptyBlockService extends AbstractBlockService
 {
     // NEXT_MAJOR: Remove this method
```

Comment:
We did remove some method here on 3.x but not removed the comment (on 3.x is also not removed, so don't do it here either). Just a reminder